### PR TITLE
Adjust to changes in arcithecture

### PR DESCRIPTION
**Pull request by RuneLind**
https://github.com/statisticsnorway/dapla-project/blob/master/doc/architecture.adoc

### CODE REVIEW

File: src/main/proto/service.proto [hunk 1]
```diff
@@ -15,34 +14,13 @@ message AccessTokenRequest {
...
     }
     Privilege privilege = 2;
-    string location = 3;
+    string path = 3; // example: '/rawdata/skatt/konto'
 }
 
 message AccessTokenResponse {
     string accessToken = 1;
     int64 expirationTime = 2;
+    string parentUri = 3;
```
Comment (bjornandre) [inline src/main/proto/service.proto]: What is parentUri? Is it the same as path?

File: src/main/proto/service.proto [hunk 2]
```diff
@@ -15,34 +14,13 @@ message AccessTokenRequest {
         WRITE = 1;
     }
     Privilege privilege = 2;
-    string location = 3;
+    string path = 3; // example: '/rawdata/skatt/konto'
```
Comment (bjornandre) [inline src/main/proto/service.proto]: Ok to use path - but location was chosen to correspond with the same location that is used in catalog service. So I suggest changing catalog service as well.

File: src/main/proto/service.proto (hunk 1 above)
Comment (RuneLind) [reply src/main/proto/service.proto]: First suggestion was physical Location. Full path to parquet data will be this parentUri + path from AccessTokenRequest

File: src/main/proto/service.proto (hunk 2 above)
Comment (RuneLind) [reply src/main/proto/service.proto]: It is changed this in catalog as well.

**Pull request merged**